Google.Api.Generator/ConfigProtos/service_config.proto

```diff
@@ -0,0 +1,327 @@
+﻿// Copyright 2016 The gRPC Authors
```

Comment:
Presumably eventually this will end up somewhere in github centrally?

Reply:
Yes.